hal/halx86/CMakeLists.txt

```diff
@@ -61,14 +61,23 @@ if(ARCH STREQUAL "i386")
     include(up.cmake)
     include(pic.cmake)
     include(xbox.cmake)
+    include(pc98.cmake)
     add_subdirectory(minihal)
 
+    remove_definitions(-DSARCH_XBOX)
```

Comment:
Hmmmm....

Reply:
Yes, our way of building x86 subarchitecture ports should be a bit revamped -- without SARCH variables, but with separate CMake targets (like `bootcd_xbox`, `livecd_pc98`, and etc.). That will make all binaries compile in x86 CI builders, but final targets would include only needed stuff.